modules/os2forms_attachment/os2forms_attachment.module

```diff
@@ -317,3 +318,12 @@ function os2forms_attachment_module_implements_alter(&$implementations, $hook) {
   }
 
 }
+
+/**
+ * Implements hook_preprocess().
+ *
+ * Add 'base_url' variable to be used by templates.
+ */
+function os2forms_attachment_preprocess(&$vars) {
+  $vars['base_url'] = Settings::get('base_url');
```

Comment:
We need some documentation on how and where to set the actual `base_url` value.

Reply:
Added documentation.